src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import duke.command.Command;
+import duke.exceptions.DirectoryNotFoundException;
+import duke.exceptions.DukeException;
+import duke.parser.Parser;
+import duke.storage.Storage;
+import duke.ui.Ui;
+
+
+
+
```

Comment:
Perhaps you can consider adding a comment for the Duke class briefly explaining what the Duke class do.